packages/core/admin/admin/src/pages/MarketplacePage/PluginCard/index.js

```diff
@@ -1,185 +1,110 @@
-            </div>
-            <div
-              onClick={e => {
-                // FIXME: dead link as we are changing the naming. Would be better to use a url comming from the api call directly
-                window.open(
-                  `https://github.com/strapi/strapi/tree/master/packages/strapi-plugin-${this.props.plugin.id}`,
-                  '_blank'
-                );
-              }}
-            >
-              {this.props.plugin.name} <i className="fa fa-external-link-alt" />
-            </div>
-          </div>
-          <div className="cardDescription">{descriptions.long}</div>
-          <div className="cardFooter" onClick={e => e.stopPropagation()}>
-            <div className="cardFooterButton">
-              <CheckPermissions permissions={adminPermissions.marketplace.install}>
-                <Button
-                  className={`${buttonClass} button`}
-                  label={buttonLabel}
-                  type="button"
-                  onClick={this.handleDownloadPlugin}
-                />
-              </CheckPermissions>
-            </div>
-            {this.props.isAlreadyInstalled ? (
-              settingsComponent
-            ) : (
-              <div className="compatible">
-                <i className={`fa fa-${this.props.plugin.isCompatible ? 'check' : 'times'}`} />
-                <FormattedMessage
-                  id={`app.components.PluginCard.compatible${
-                    this.props.plugin.id === 'support-us' ? 'Community' : ''
-                  }`}
-                />
-              </div>
-            )}
-          </div>
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import { Typography } from '@strapi/design-system/Typography';
+import { Button } from '@strapi/design-system/Button';
+import { LinkButton } from '@strapi/design-system/LinkButton';
+import { Flex } from '@strapi/design-system/Flex';
+import ExternalLink from '@strapi/icons/ExternalLink';
+import Duplicate from '@strapi/icons/Duplicate';
+
+const PluginLogo = styled.img`
+  display: block;
+  width: 64px;
+  height: 64px;
+  object-fit: contain;
+  border-radius: 6px;
+`;
+
+const PluginDescription = styled.p`
+  /* stylelint-disable value-no-vendor-prefix, property-no-vendor-prefix */
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  /* stylelint-enable value-no-vendor-prefix, property-no-vendor-prefix */
+  overflow: hidden;
+`;
+
+const PluginCard = ({ plugin }) => {
+  const { id, attributes } = plugin;
+  const { formatMessage } = useIntl();
+
+  return (
+    <Box
+      paddingLeft={6}
+      paddingRight={6}
+      paddingTop={4}
+      paddingBottom={4}
+      hasRadius
+      background="neutral0"
+      key={id}
+      shadow="tableShadow"
+      style={{ height: '100%' }}
```

Comment:
Are you sure you need to add inline style?

Reply:
I couldn't get rid of this inline style, otherwise cards from the same row would have different heights